_layouts/page.html

```diff
@@ -2,7 +2,7 @@
 layout: default
 ---
 <div class="container">
-  <article class="post">
+  <article class="post {% if page.url == '/conference/' %}extended{% endif %}">
```

Comment:
Should the banner persist on all pages?
There's an issue with the header overlapping content on`/about` and other pages.

Reply:
I was hoping to have it persist! That sorta makes sense, right?

Reply:
Okay fixed!